### PR TITLE
Generate lib for arm64

### PR DESCRIPTION
**Pull request by ferjm**

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -36,10 +38,11 @@ do
...
   cd $GST_LIB
-  sed -i 's?libdir=.*?libdir='`pwd`'?g' pkgconfig/*
-  sed -i 's?.* -L${.*?Libs: -L${libdir} -lgstreamer_android?g' pkgconfig/*
-  sed -i 's?Libs:.*?Libs: -L${libdir} -lgstreamer_android?g' pkgconfig/*
-  sed -i 's?Libs.private.*?Libs.private: -lgstreamer_android?g' pkgconfig/*
+  sed -i -e 's?libdir=.*?libdir='`pwd`'?g' pkgconfig/*
+  sed -i -e 's?.* -L${.*?Libs: -L${libdir} -lgstreamer_android?g' pkgconfig/*
+  sed -i -e 's?Libs:.*?Libs: -L${libdir} -lgstreamer_android?g' pkgconfig/*
+  sed -i -e 's?Libs.private.*?Libs.private: -lgstreamer_android?g' pkgconfig/*
+  rm -rf pkgconfig/*pc-e*
```
Comment (ferjm) [inline build.sh:45]: This is reverting #5 cause I found that the script won't work properly on MacOS otherwise.

**Review by asajeffrey — COMMENTED** [review]
Should there really be checked in files in `out`? Apart from that, lgtm.

**Comment by ferjm** [comment]
The `out` folder is there for historical reasons (Servo started getting the files directly from GH instead of S3). I think there's still at least [one extra consumer](https://github.com/servo/libgstreamer_android_gen/issues/2#issuecomment-427551575), but maybe it's time to get rid of the `out` folder, as the binaries are on git's history anyway. cc @alanpoon

**Comment by asajeffrey** [comment]
I'll leave it up to you whether to delete it as part of this PR. It would be nice if we could at least rename it to something less confusing, like support/ARCH/gstreamer. You can r=me.

**Comment by Manishearth** [comment]
I undid the change to remove the out/ directory -- old servo commits reference these files directly without using a hash as a permalink, which makes this hard to bisect. We can revert this once I'm done bisecting (but perhaps it's best left in and marked as a deprecated folder)

**Pull request merged**